src/components/ActionBar/src/InlineActionBar.vue

```diff
@@ -26,14 +44,18 @@ export default {
 	props: {
 		position: {
 			type: String,
-			default: 'static',
+			default: 'absolute',
 			validator: (position) => ['static', 'relative', 'absolute', 'fixed'].includes(position),
 		},
 		hideOn: {
 			type: String,
 			default: 'none',
 			validator: (hideOn) => ['none', 'mobile', 'desktop'].includes(hideOn),
 		},
+		flush: {
```

Comment:
When would you use flush? Typically we'd used that to adjust CSS padding, but this is doing some conditional rendering?

Reply:
It's necessary in the scenario where `InlineActionBar` is used by `ActionBarLayer`. Well, I guess I could try refactoring `InlineActionBar` into an `InlineActionBar` and an `AtomicActionBar`. That might be better.

Reply:
I did the refactor.